src/convert.rs

```diff
@@ -798,6 +815,12 @@ impl From<TransactionError> for tx_by_addr::TransactionError {
                 TransactionError::WouldExceedAccountDataTotalLimit => {
                     tx_by_addr::TransactionErrorType::WouldExceedAccountDataTotalLimit
                 }
+                TransactionError::InsufficientFundsForRent { .. } => {
```

Comment:
Make it consistent with validator code:

                TransactionError::DuplicateInstruction(_) => {
                    tx_by_addr::TransactionErrorType::DuplicateInstruction
                }
                TransactionError::InsufficientFundsForRent { .. } => {
                    tx_by_addr::TransactionErrorType::InsufficientFundsForRent
                }